plugins/nlp/nlp.cpp

```diff
@@ -68,6 +71,24 @@ namespace nlp {
         tgt = (char *) CTXMALLOC(parentCtx, tgtLen);
         memcpy_iflen(tgt, sso.str().c_str(), tgtLen);
     }
+
+    ECL_NLP_API void ECL_NLP_CALL AnalyzeTextU(unsigned & tgtLen, UChar * & tgt, size32_t anaLen, const char * ana, unsigned txtLen, UChar const * txt)
+    {
+        {
+            CriticalBlock block(cs);
+            if (nlpEng == NULL) {
+                nlpEng = new NLPEng();
+            }
+        }
+        UnicodeString const in(txt, txtLen);
```

Comment:
minor: The following is more efficient since it avoids cloning the string:

    UnicodeString const in(false, txt, txtLen);